docs/docs/en/contribute/join/become-a-committer.md

```diff
@@ -1,11 +1,170 @@
+The PMC regularly adds new committers from the active contributors, based on their contributions to Dolphinscheduler. The qualifications for new committers include:
 
-More details could be found [here](https://community.apache.org/contributors/).
+Sustained contributions to Dolphinscheduler: Committers should have a history of major contributions to Dolphinscheduler. An ideal committer will have contributed broadly throughout the project, and have contributed at least one major component where they have taken an “ownership” role. An ownership role means that existing contributors feel that they should run patches for this component by this person.
+
+Quality of contributions: Committers more than any other community member should submit simple, well-tested, and well-designed patches. In addition, they should show sufficient expertise to be able to review patches, including making sure they fit within Dolphinscheduler’s engineering practices (testability, documentation, API stability, code style, etc). The committership is collectively responsible for the software quality and maintainability of Dolphinscheduler. Note that contributions to critical parts of Dolphinscheduler, like its core and SQL modules, will be held to a higher standard when assessing quality. Contributors to these areas will face more review of their changes.
+
+Community involvement: Committers should have a constructive and friendly attitude in all community interactions. They should also be active on the dev and user list and help mentor newer contributors and users. In design discussions, committers should maintain a professional and diplomatic approach, even in the face of disagreement.
+
+## Nominate new committer
+
+In Dolphinscheduler, **new committer nomination** could only be officially started by existing PMC members. If a new committer feels that he/she is qualified, he/she should contact any existing PMC member and discuss. If this is agreed among some members of the PMC, the process will kick off.
+
+The following steps are recommended (to be initiated only by an existing 2PMC member):
+1. Send an email titled `[DISCUSS] Promote xxx as new committer` to `private@dolphinscheduler.apache.org`. List the important contributions of the candidate,
+   so you could gather support from other PMC members for your proposal.
+2. Keep the discussion open for more than 3 days but no more than 1 week, unless there is any express objection or concern.
+3. If the PMC generally agrees to the proposal, send an email titled `[VOTE] Promote xxx as new committer` to `private@dolphinscheduler.apache.org`.
+4. Keep the voting process open for more than 3 days, but no more than 1 week. Consider the result as `Consensus Approval` if there are three +1 votes and
```

Comment:
A positive result is at least 3 + 1 votes with NO vetoes, see: https://community.apache.org/newcommitter.html

Reply:
Resolved.